addons/skin.estouchy/language/resource.language.en_gb/strings.po

```diff
@@ -78,22 +78,27 @@ msgctxt "#31016"
 msgid "Albums"
 msgstr ""
 
+#: /xml/DialogPlayerProcessInfo.xml
```

Comment:
Please add verbal descriptions for the labels to make the life of our translaters easier.

`#. Label for ...`

Reply:
done

Reply:
> done

I don't see any descriptions for the new/changed labels in the now merged PR!